tests/example_tests/test_libuserd_exhaustive.py

```diff
@@ -312,6 +338,179 @@ def get_userd_values(userd_part_var_timestep_cache, timestep, part_id, var_id):
     return userd_values
 
 
+def convert_element_map(elemental_map_userd):
+    """
+    Converts a dictionary with element strings as keys to a dictionary with element numbers as keys.
+
+    Args:
+    elemental_map_userd (dict): A dictionary with element strings as keys and element counts as values.
+
+    Returns:
+    dict: A new dictionary with element numbers as keys and element counts as values.
+    """
+    # Local mapping inside the function to convert element strings to numbers
+    element_string_to_int_map = {
```

Comment:
@vthiagar this is a constant, it can be defined outside of the function, top of the file, and used here. So it is not defined every time the function is called.
Usually constants names in Python are upper case

Reply:
Done